docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -69,6 +69,18 @@ Currently, only the `Alias`, `WantedBy` and `RequiredBy` keys are supported.
 words `WantedBy=other.service`, not `WantedBy=other.container`. The same is
 true for other kinds of dependencies, too, like `After=other.service`.
 
+### Debugging unit files
+
+After placing the unit file in a unit search path (mentioned above), you can enable it with `systemctl enable`.
```

Comment:
This comment is incorrect. `systemctl enable` will always fail. The reason for the failure is that the `.service` file is generated by Quadlet and as a result considered by systmed as transient which systemd does not allow enabling.

We have opened a ticket with systemd (https://github.com/systemd/systemd/issues/28006) in order to find a way to allow enabling Quadlet generated services, but have yet to receive any response

Reply:
Thank you. Fixed.

I was already using the `[Install]` section, just messed up while writing the doc.